playbooks/common/openshift-cluster/additional_config.yml

```diff
@@ -54,3 +54,5 @@
   - role: openshift_registry
     registry_volume_claim: "{{ openshift.hosted.registry.storage.volume.name }}-claim"
     when: deploy_infra | bool and attach_registry_volume | bool
+  - role: openshift_metrics
+    when: use_openshift_metrics | bool
```

Comment:
@abutcher can you make recommendations to tie this into the persistent storage work...  then this condition could be:
```
when: deploy_infra | bool and attach_metrics_volumes | bool
```